delivery.yaml

```diff
@@ -0,0 +1,42 @@
+build_steps:
```

Comment:
wondering, if it would make sense to put the actual commands into Makefile tasks, and only trigger the task from the yaml. That way it may be more accessible and testable for those who don't have a CDP build environment.